CHANGELOG.md

```diff
@@ -1,3 +1,109 @@
+### citus v6.0.0 (November 4, 2016) ###
+
+* Adds compatibility with PostgreSQL 9.6, now the recommended version
+
+* Removes the `pg_worker_list.conf` file in favor of a `pg_dist_node` table
+
+* Removes the `\stage` command and corresponding csql binary in favor of `COPY`
+
+* Removes `copy_to_distributed_table` in favor of first-class `COPY` support
+
+* Adds support for multiple DDL statements within a transaction
+
+* Adds support for certain foreign key constraints
+
+* Adds limited support for `INSERT INTO ... SELECT FROM` commands
+
+* Adds support for the `TRUNCATE` command
+
+* Adds support for `HAVING` clauses in `SELECT` queries
+
+* Adds support for `EXCLUDE` constraints which include the partition column
+
+* Adds support for system columns in queries (`tableoid`, `ctid`, etc.)
+
+* Adds support for relation name extension within `INDEX` definitions
+
+* Adds support for no-op `UPDATE`s of the partition column
+
+* Adds several general-purpose utility UDFs to aid in Citus maintenance
+
+* Adds `master_expire_table_cache` UDF to forcibly expire cached shards
+
+* Extends the router planner to handle more queries
+
+* Parallelizes the processing of DDL commands which affect distributed tables
+
+* Adds support for repartition jobs using composite or custom types
+
+* Enhances object name extension to handle long names and large shard counts
+
+* Parallelizes the `master_modify_multiple_shards` UDF
+
+* Changes distributed table creation to error if the target table is not empty
+
+* Changes the `pg_dist_shard.logicalrelid` column from an `oid` to `regclass`
+
+* Adds a true `placementid` column to `pg_dist_shard_placement`
+
+* Removes the `pg_dist_shard.shardalias` distribution metadata column
+
+* Adds `pg_dist_partition.repmodel` to track tables using streaming replication
+
+* Adds internal infrastructure to take snapshots of distribution metadata
+
+* Addresses the need to invalidate prepared statements on metadata changes
+
+* Adds a `mark_tables_colocated` UDF for denoting pre-6.0 manual colocation
+
+* Fixes a bug affecting prepared statement execution within PL/pgSQL
+
+* Fixes a bug affecting `COPY` commands using composite types
+
+* Separates worker and master job temporary folders
+
+* Eliminates race condition between distributed modification and repair
+
+* Relaxes the requirement that shard repairs also repair colocated shards
+
+* TODO: What about #785 ?
+
+* TODO: #840
+
+* TODO: #865
+
+* TODO: #864
```

Comment:
Maybe as described "adds the required distributed transaction API for executing commands on workers within a transaction."? Could add note again as preparation for future MX changes.